inc/telemetry.class.php

```diff
@@ -343,7 +343,9 @@ public static final function generateRegistrationUuid() {
    public static function getViewLink() {
       global $CFG_GLPI;
 
-      $out = "<a id='view_telemetry' href='{$CFG_GLPI['root_doc']}/ajax/telemetry.php'>" . __('See what would be sent...') . "</a>";
+      $out = "<a id='view_telemetry' href='{$CFG_GLPI['root_doc']}/ajax/telemetry.php' class='btn btn-sm btn-info mt-2'>
```

Comment:
Text in Telemetry dialog is not readable.

![image](https://user-images.githubusercontent.com/33253653/133083698-7047ce90-3c81-4374-b7f7-ac0067477547.png)

Reply:
Fixed.